tests/func/test_remove.py

```diff
@@ -32,10 +32,29 @@ def test_remove(tmp_dir, scm, dvc, run_copy, remove_outs):
     assert not (tmp_dir / ".gitignore").exists()
 
 
+def test_remove_file_as_target(tmp_dir, scm, dvc):
+    dvc.run(
+        name="my",
+        cmd='echo "hello" > foo && echo "hello" > bar',
+        deps=[],
+        outs=["foo", "bar"],
+    )
+
+    assert "/foo" in get_gitignore_content()
+    assert "/bar" in get_gitignore_content()
```

Comment:
This is not a part of `remove`, so we can assume it works (or tested for `dvc.run`).